docs/src/pages/components/autocomplete/Virtualize.js

```diff
@@ -41,7 +56,10 @@ function useResetCache(data) {
 // Adapter for react-window
 const ListboxComponent = React.forwardRef(function ListboxComponent(props, ref) {
   const { children, ...other } = props;
-  const itemData = React.Children.toArray(children);
+  const itemData = children.reduce((acc, item) => {
+    acc.push(item);
+    return acc.concat(item.children);
+  }, []);
```

Comment:
Let's rewrite this with `forEach`. `reduce` is not actually used here since the accumulator is mutated and we don't really need `reduce` here anyway. It just makes the demo harder to follow